deployments/ism/nexus/config.yaml

```diff
@@ -0,0 +1,71 @@
+# yaml-language-server: $schema=./core-deployments.schema.json
+---
+description: ISM config for Nexus
+timestamp: 2023-09-23T16:00:00.000Z
```

Comment:
Is this timestamp right?

Reply:
No, but I believe currently unused? I just had it copy-pasted